### DISCUSSION TITLE
Is there someone that runs Firefly through NGINX ?

### DISCUSSION THREAD

**Discussion by lucaslgr** [firefly-iii/firefly-iii#8794] [General]
I want to run Firefly and another Web application in the same server. So, to do that I need NGINX as a proxy to each application, but I have some doubts about how I need to configure my docker-compose.yml and my nginx.conf. Has anyone done something like this and could show me an example?

**Comment by JC5** [comment]
I know there are a lot of examples when you search for them, is there anything specific you're looking for?

**Reply by lucaslgr** [reply]
I just want to run firefly through NGINX instead of the application directly. But I'm running all in Docker.

**Reply by lucaslgr** [reply]
I achieved it using this container, using [nginx-proxy](https://hub.docker.com/r/nginxproxy/nginx-proxy), that has an algorithm that update the proxy configurations in NGINX using `docker-gen`.
